src/commands/deployment/get.ts

```diff
@@ -146,6 +159,40 @@ export const getDeployments = (
   });
 };
 
+/**
+ * Gets a list of deployments for the specified filters every 5 seconds
+ * @param outputFormat output format: normal | wide | json
+ * @param environment release environment, such as Dev, Staging, Prod etc.
+ * @param imageTag docker image tag name
+ * @param p1Id identifier of the first build pipeline (src to ACR)
+ * @param commitId commit Id into the source repo
+ * @param service name of the service that was modified
+ * @param deploymentId unique identifier for the deployment
+ */
+export const watchGetDeployments = (
+  outputFormat: OUTPUT_FORMAT,
+  environment?: string,
+  imageTag?: string,
+  p1Id?: string,
+  commitId?: string,
+  service?: string,
+  deploymentId?: string
+): void => {
+  const timeInterval = 5000;
```

Comment:
nit: Can we add this `timeInterval` to the top of the file so it's easy to find and edit?